New version: StyledStrings v1.0.3

JuliaRegistrator opened the pull request:
- Registering package: StyledStrings
- Repository: https://github.com/JuliaLang/StyledStrings.jl
- Created by: @tecosaur
- Version: v1.0.3
- Commit: 1b38e5376de0194e7df012bc0d0f99b4df7f5029
- Reviewed by: @tecosaur
- Reference: https://github.com/JuliaLang/StyledStrings.jl/commit/1b38e5376de0194e7df012bc0d0f99b4df7f5029#commitcomment-145780032
- Description: Write with ✨ style ✨ and ease
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6b712d300ff8b45e685f255eba655c5b56bceeece76b8cc287406bf647a78cec801a293f1e863ecf456ec7249d21b01d6524a3b0dd64a6c240f462bcbecbefa25fb5af448adb831e442f78532f9f7a2248675deb79799adc4d82bfa8e61dfbbd8b40b0341e2528f0ed7e13bba6a8141b63c3620f892036907965ce82a587ecd11bdcc5293e78081977f0b21165459690240daaab720b95db7d17481068447231054ce46948519ed77db7db914b508f0e7 -->

tecosaur commented:
Actually, let's block this for a sec, I might want to tweak one or two more things

JuliaRegistrator committed:
New version: StyledStrings v1.0.3

UUID: f489334b-da3d-4c2e-b8f0-e476e12c162b
Repo: https://github.com/JuliaLang/StyledStrings.jl.git
Tree: 711c9650010c95814911c2005ea04e70e70e65ed

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
